shared/crypto/libp2p.go

```diff
@@ -0,0 +1,23 @@
+package crypto
+
+import (
+	"encoding/hex"
+
+	"github.com/libp2p/go-libp2p/core/crypto"
+)
+
+// NewLibP2PPrivateKey converts a hex-encoded ed25519d key
+// string into a libp2p compatible Private Key.
+func NewLibP2PPrivateKey(hexString string) (crypto.PrivKey, error) {
+	keyBytes, err := hex.DecodeString(hexString)
```

Comment:
NIT: We got into the pattern of using `keyBz` for bz

Reply:
Cool, is this a popular convention?